src/api/content-strings/fields/insurance/your-buyer/index.ts

```diff
@@ -99,7 +99,7 @@ export const YOUR_BUYER_FIELDS = {
     },
   },
   [FAILED_PAYMENTS]: {
-    LABEL: 'Has the buyer ever failed to pay you on time?',
+    HINT: 'This is when an invoice has still not been paid 30 days or more after the agreed payment date.',
```

Comment:
just a suggestion, worth having 30 as a constant?

Reply:
I thought about this and don't think its worth it since this is purely for some visual help text for a user. If there was any business logic involved we should definitely use a constant.